### PR TITLE
Feature 3

### PR DESCRIPTION
**Pull request by EduardTymchenko**

### CODE REVIEW

File: src/main/java/com/example/clinic/domain/Clinic.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+import java.util.List;
+
+public class Clinic {
+    private long id;
+    private String name;
+    private String location;
+    private String phone;
+    private boolean isPublic;
+    private boolean isInsurance;
```
Comment (Andriireva) [inline src/main/java/com/example/clinic/domain/Clinic.java]: I would change name to "hasInsurance"

File: src/main/java/com/example/clinic/domain/Clinic.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package com.example.clinic.domain;
+
+import java.util.List;
+
+public class Clinic {
+    private long id;
+    private String name;
+    private String location;
+    private String phone;
+    private boolean isPublic;
```
Comment (Andriireva) [inline src/main/java/com/example/clinic/domain/Clinic.java]: I suggest to change type to enum where enum can be PRIVATE, PUBLIC

File: src/main/java/com/example/clinic/domain/Service.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
+package com.example.clinic.domain;
+
+
+public class Service {
+    private long id;
+    private String name;
+    private float fee;
+    private int percentInsurance;
+    private ThreadLocal startTime;
+    private ThreadLocal endTime;
```
Comment (Andriireva) [inline src/main/java/com/example/clinic/domain/Service.java]: Lets simplify it to String where it can be "1h" or "1d" or "30m".

**Review by Andriireva — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/clinic/domain/Service.java (hunk 3 above)
Comment (EduardTymchenko) [reply src/main/java/com/example/clinic/domain/Service.java]: simplified to String

**Review by Andriireva — APPROVED** [review]

**Pull request merged**